jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditor.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.SimpleToolWindowPanel;
+import org.jdesktop.swingx.JXDatePicker;
+import javax.swing.ButtonGroup;
+import javax.swing.JButton;
+import javax.swing.JComboBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+
+public class QueryEditor {
+    public JRadioButton absoluteTimeRadioButton;
+    public JRadioButton relativeTimeRadioButton;
+    public JRadioButton searchTerm;
+    public JTextField querySearchTerm;
+    public JRadioButton queryLogGroupsRadioButton;
+    private JButton saveQueryButton;
+    private JButton retrieveSavedQueriesButton;
```

Comment:
for readability, don't bind objects to the class if they aren't going to be used anywhere